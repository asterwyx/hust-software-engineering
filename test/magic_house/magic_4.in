preset user AQS
step 3 #A
n
step 63 #Q
2 #choose A

dump