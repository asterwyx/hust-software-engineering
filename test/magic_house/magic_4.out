user AQS

fund A 10000
fund Q 10000
fund S 10000

credit A 0
credit Q 0
credit S 0

userloc A 3 2
userloc Q 63 0
userloc S 0 0

nextuser S